tests/Predis/Command/Redis/GET_Test.php

```diff
@@ -90,6 +90,18 @@ public function testReturnsNullOnNonExistingKeys(): void
         $this->assertNull($redis->get('foo'));
     }
 
+    /**
+     * @group connected
+     */
+    public function testReturnsStatusInTransaction(): void
+    {
+        $redis = $this->getClient();
+
+        $this->assertSame(0, $redis->exists('foo'));
+        $this->assertInstanceOf('Predis\Response\Status', $redis->multi());
+        $this->assertInstanceOf('Predis\Response\Status', $redis->get('foo'));
```

Comment:
Ah, I see. You're getting a `Status` response when opening a `MULTI` transaction. Theoretically _all_ commands then can return `Error|Status`, which I don't really want to document TBH.

Reply:
Sadly that is not true, we are not using `MULTI` anywhere and still occasionally getting the Status responses… which seems possible according to Google search when the server is under high load…

The reason I used `MULTI` here was that it is the only way known to me to reproduce the issue in a deterministic way…

On the other hand I completely understand why you don’t want to add it ewerywhere yet when relying on the code of the library and using the static analysis tools this creates a great pain:
- not aware that not documented types can be returned
- even when aware of it you need to mute static analysis warning about dead code… because it relies on the php doc comment anyway in this case…

So what is your proposal to proceed this further?

Reply:
I don't know, but adding it only for a single command `GET` without a proper test to recreate it seems too dodgy.

I'd be open to adding it to all (most) methods to be more accurate, but this requires work and tests which I don't have the bandwidth for right now.

Reply:
> I'd be open to adding it to all (most) methods to be more accurate, but this requires work and tests which I don't have the bandwidth for right now. If it's important to you please extend the PR 👍

Will see what I can do :-)